openpype/hosts/max/plugins/publish/extract_max_scene_raw.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pyblish.api
+from openpype.pipeline import (
+    publish,
+    OptionalPyblishPluginMixin
+)
+from pymxs import runtime as rt
+from openpype.hosts.max.api import (
+    maintained_selection,
+    get_all_children
+)
+
+
+class ExtractMaxSceneRaw(publish.Extractor,
+                         OptionalPyblishPluginMixin):
+    """
+    Extract Raw Max Scene with SaveSelected
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.2
+    label = "Max Scene(Raw)"
```

Comment:
Going from the screenshot for toggling on/off these extractor it might be nicest to do `Extract Max Scene (Raw)` as a label?
```suggestion
    label = "Extract Max Scene (Raw)"
```